tests/conftest.py

```diff
@@ -24,68 +23,37 @@
 import app
 from core.authentication import AuthenticationHelper
 
-
-class Caption:
-    def __init__(self, text, highlights=None, additional_properties=None):
-        self.text = text
-        self.highlights = highlights or []
-        self.additional_properties = additional_properties or {}
-
-
-class AsyncPageIterator:
-    def __init__(self, data):
-        self.data = data
-
-    def __aiter__(self):
-        return self
-
-    async def __anext__(self):
-        if not self.data:
-            raise StopAsyncIteration
-        return self.data.pop(0)  # This should be a list of dictionaries.
-
-
-class AsyncSearchResultsIterator:
-    def __init__(self):
-        self.data = [
-            [
-                {
-                    "sourcepage": "Benefit_Options-2.pdf",
-                    "sourcefile": "Benefit_Options.pdf",
-                    "content": "There is a whistleblower policy.",
-                    "embeddings": [],
-                    "category": None,
-                    "id": "file-Benefit_Options_pdf-42656E656669745F4F7074696F6E732E706466-page-2",
-                    "@search.score": 0.03279569745063782,
-                    "@search.reranker_score": 3.4577205181121826,
-                    "@search.highlights": None,
-                    "@search.captions": [Caption("Caption: A whistleblower policy.")],
-                },
-            ]
-        ]
-
-    def __aiter__(self):
-        return self
-
-    async def __anext__(self):
-        if not self.data:
-            raise StopAsyncIteration
-        return AsyncPageIterator(self.data.pop(0))
-
-    def by_page(self):
-        return self
+from .mocks import (
```

Comment:
I moved most mocks into mocks.py because conftest.py was getting hard to wade through.